crates/router/src/core/payment_methods/vault.rs

```diff
@@ -551,3 +568,140 @@ pub async fn delete_tokenized_data(
             .attach_printable(format!("Got 4xx from the basilisk locker: {err:?}")),
     }
 }
+
+// ********************************************** PROCESS TRACKER **********************************************
+#[cfg(feature = "basilisk")]
+pub async fn add_tokenize_data_task(
+    db: &dyn db::StorageInterface,
+    lookup_key: &str,
+    pm: enums::PaymentMethod,
+    runner: &str,
+) -> RouterResult<storage::ProcessTracker> {
+    let task = "DELETE_TOKENIZE_DATA";
+    let current_time = common_utils::date_time::now();
+    let tracking_data = serde_json::to_value(storage::TokenizeCoreWorkflow {
+        lookup_key: lookup_key.to_owned(),
+        pm,
+    })
+    .into_report()
+    .change_context(errors::ApiErrorResponse::InternalServerError)
+    .attach_printable_lazy(|| format!("unable to convert into value {:?}", lookup_key))?;
+
+    let schedule_time = get_delete_tokenize_schedule_time(db, &pm, 0)
+        .await
+        .into_report()
+        .change_context(errors::ApiErrorResponse::InternalServerError)
```

Comment:
for this case we need to proceed with a default value and not as error.

Reply:
sure